packages/findlib/findlib.1.8.1+dune2/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer:   "Anil Madhavapeddy <anil@recoil.org>"
+homepage:     "https://github.com/dune-universe/lib-findlib"
+bug-reports:  "https://github.com/dune-universe/lib-findlib/issues"
+dev-repo: "git+https://github.com/dune-universe/lib-findlib.git"
+build: [
+  [ "env" "FINDLIB_PREFIX=%{lib}%" "dune" "build" "-p" name "-j" jobs ]
```

Comment:
do we really need to have `FINDLIB_PREFIX` here? We are not building this with opam anyway...